actionview/test/template/date_helper_test.rb

```diff
@@ -472,7 +472,7 @@ def test_select_year
 
   def test_select_year_with_empty_hash_value_and_no_start_year
     expected = +%(<select id="date_year" name="date[year]">\n)
-    expected << %(<option value="2014">2014</option>\n<option value="2015">2015</option>\n<option value="2016">2016</option>\n<option value="2017">2017</option>\n<option value="2018">2018</option>\n)
+    expected << %(<option value="2015">2015</option>\n<option value="2016">2016</option>\n<option value="2017">2017</option>\n<option value="2018">2018</option>\n)
```

Comment:
I haven't used `select_year` DateHelper yet but the test cases were failing on the current PR so I debugged the issue and found that these test cases are failing because of the new year started(i.e: 2019 -> 2020)

`select_year` uses `middle_year`(i.e: Date.today.year -> `2020`. ref: https://github.com/rails/rails/blob/master/actionview/lib/action_view/helpers/date_helper.rb#L835) when `year` or `@datetime` is zero. So the `start_year`  for this test case would be `2015` and not `2014`.

These test cases would fail next year as well. So let me know if any suggestions on tackling it in better way. Thanks

Reply:
Any chance to use `travel_to` in this test?

Reply:
should be fixed by https://github.com/rails/rails/pull/38136